runtime/reconcile/doc.go

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright 2022 The Flux authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package reconcile provides helpers for the reconciliation. It helps finalize
```

Comment:
```suggestion
// Package reconcile provides helpers for handling reconciliations. They help finalize
```

Reply:
It's not really for handling reconciliations. Accepted the rest of the suggestion.